openeo-geotrellis/src/main/scala/org/openeo/geotrellis/udf/Udf.scala

```diff
@@ -35,7 +36,7 @@ object Udf {
   private def _createSharedInterpreter(): SharedInterpreter = {
     if (!isInterpreterInitialized) {
       val config = new JepConfig()
-      config.setRedirectOutputStreams(true)
+      //config.setRedirectOutputStreams(true)
```

Comment:
We should use the new redirectStdout, redirectStdErr methods here to avoid losing UDF logging.